.gitignore

```diff
@@ -1,5 +1,6 @@
 node_modules/
 package-lock.json
+examples/**/yarn.lock
```

Comment:
Unless I'm missing something, we'd want to keep the lock file for the examples too, wouldn't we?

Reply:
👍